### PR TITLE
Adding jasmine-expect TS definitions

### PR DESCRIPTION
**Pull request by PillowPillow**
Added TypeScripts definitions for [`JamieMason/Jasmine-Matchers`](https://github.com/JamieMason/Jasmine-Matchers) module.

### CODE REVIEW

**Comment by vvakame** [comment]
*jasmine-expect/jasmine-expect.d.ts*

check list

* [ ] is collect [naming convention](http://definitelytyped.org/guides/contributing.html#naming-the-file)?
  * https://www.npmjs.com/package/jasmine-expect
  * http://bower.io/search/?q=jasmine-expect
  * others?
* [ ] has a [test file](http://definitelytyped.org/guides/contributing.html#tests)? (jasmine-expect/jasmine-expect-tests.ts or others)
* [ ] pass the Travic-CI test?

File: jasmine-expect/jasmine-expect.d.ts [hunk 1]
```diff
@@ -0,0 +1,127 @@
+// Type definitions for jasmine-expect v2.0.0-beta1
+// Project: https://github.com/JamieMason/Jasmine-Matchers
+// Definitions by: Nicolas Gaignoux <https://github.com/PillowPillow>
+// Definitions: https://github.com/borisyankov/DefinitelyTyped
+
+/*
+Typings 2015 Nicolas Gaignoux
```
Comment (vvakame) [inline jasmine-expect/jasmine-expect.d.ts]: License is here. https://github.com/borisyankov/DefinitelyTyped/blob/master/LICENSE
This is may not be explicitly shown in individual files.

**Comment by mhegazy** [comment]
Thank you for your contribution. 

Can you please fix the failing Travis CI tests and update the PR!

**Comment by RyanCavanaugh** [comment]
Hello and thank you for your contribution!

Due to an excessively long queue of pull requests that have become stale, we are "declaring bankruptcy" and closing all PRs opened before May 1, 2016. If you'd still like to merge this code in, please open a new PR that has been merged and rebased with the `master` branch.

Going forward, we are committing to review or merge all PRs on a regular basis so this bankruptcy will not occur again. We apologize for the incovenience and hope you will continue to contribute to DefinitelyTyped in the future.

Thanks
The TypeScript and DefinitelyTyped teams

**Pull request closed without merging**